src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -364,6 +364,11 @@ private static Lazy<HashSet<string>> CreateInternalsVisibleToMap(IAssemblySymbol
                     }
 
                     var value = (string)typeNameConstant.Value;
+                    if (string.IsNullOrWhiteSpace(value))
```

Comment:
If we're strictly fixing the NRE then, yes, you're right. Even a leading comma would have previously resulted in String.Empty going in the map. As I mentioned in reply to your other comment, I was thinking of adding more validation here, but didn't. In that vein,  I'll change this check to a simple null check.